workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -59,31 +58,32 @@ export const mapProcessInstanceToDetails = (
   };
 };
 
-const useStyles = makeStyles(_ => ({
-  topRowCard: {
-    height: '20rem',
-  },
-  middleRowCard: {
-    height: '20rem',
-    overflow: 'auto',
-    wordBreak: 'break-word',
-  },
-  bottomRowCard: {
-    minHeight: '40rem',
-    height: '100%',
-  },
-  autoOverflow: { overflow: 'auto' },
-  recommendedLabelContainer: {
-    display: 'flex',
-    alignItems: 'center',
-    whiteSpace: 'nowrap',
-  },
-  recommendedLabel: { margin: '0 0.25rem' },
-}));
-
 export const WorkflowInstancePageContent: React.FC<{
   assessedInstance: AssessedProcessInstanceDTO;
 }> = ({ assessedInstance }) => {
+  const height = assessedInstance.assessedBy ? '21rem' : '18rem';
+  const useStyles = makeStyles(_ => ({
```

Comment:
It's not good practice to define a hook inside a component
See [here](https://github.com/LiorSoffer/rhdh-plugins/blob/updateInstancePage/workspaces/orchestrator/plugins/orchestrator/src/components/ui/OverrideBackstageTable.tsx#L29) an example how to pass properties to makeStyles